README.md

```diff
@@ -1 +1,179 @@
-# Capstone-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>My README File</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Capstone Project] <a name="Foursquare Annual Convention 2023 Namibia"></a>
+
+**[Foursquare Annual Convention]** is an online website for an Annual Convetion conference that happens every year at Foursquare Gospel church in Namibia. This same event also happens at Foursquare Gospel Churches in other countries world wide. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+HTML, CSS AND JAVASCRIPT
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
```

Comment:
- [ ] Did you make use of PostgreSQL for your HTML & CSS capstone module?. I kindly suggest that you remove react here and replace it with HTML, CSS, and JavaScript because those are the front end technologies you used for your project.